k8s/base/postgres/secret.yaml

```
@@ -0,0 +1,14 @@
+metadata:
+  name: postgres-secret
+  labels:
+    app: postgres
+    component: database
+type: Opaque
+data:
+  # Base64 encoded values
+  # Default values for development - CHANGE FOR PRODUCTION
+  username: cG9zdGdyZXM=  # postgres
+  password: cG9zdGdyZXM=  # postgres
+  database: bmVzdGpzX3RlbXBsYXRl  # nestjs_template
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)